tests/ttnn/profiling/ops_for_profiling.py

```diff
@@ -2160,13 +2160,13 @@ def assign_unary(x):
         "layout": "ROW_MAJOR",
         "num_repeats": 2,
     },
-    {
-        "op": argmax_dim_2,
-        "name": "ttnn.argmax_dim_2",
-        "shape_func": argmax_shape_func,
-        "layout": "ROW_MAJOR",
-        "num_repeats": 2,
-    },
+    # {
+    #     "op": argmax_dim_2,
```

Comment:
Same here, any reason this config is commented out?

Reply:
this was never functional -- dim was defaulted to None

Reply:
Let's not comment out code, just remove it if it's not needed.